src/main/java/io/aiven/kafka/connect/http/sender/HttpRequestBuilder.java

```diff
@@ -33,17 +33,13 @@
         final var httpRequest = HttpRequest.newBuilder(config.httpUri())
                 .timeout(Duration.ofSeconds(config.httpTimeout()));
         config.getAdditionalHeaders().forEach(httpRequest::header);
-        return httpRequest;
-    };
-
-    HttpRequestBuilder AUTH_HTTP_REQUEST_BUILDER = config -> {
-        final var requestBuilder =
-                DEFAULT_HTTP_REQUEST_BUILDER.build(config)
-                        .header(HEADER_AUTHORIZATION, config.headerAuthorization());
         if (config.headerContentType() != null) {
```

Comment:
I know this wasn't part of your PR, how would you feel adding an empty String check here?
```suggestion
        if (config.headerContentType() != null && !config.headerContentType().equals("")) {
```

Reply:
Will do, would "" suffice or should we use something like `isBlank()`?

Reply:
We could use the `isBlank()` method in `String` yes :)

Reply:
Actually can/should we use the config validator approach here and validate this at config time, so that the user is warned at the validation step?

Reply:
That would make sense yes!

Reply:
Added NonBlankStringValidator